lib/key-value-manager/key-value-manager.abstract.ts

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs';
+import * as process from 'process';
+
+import { JsonValue } from 'types/general';
+import { isDefined } from 'types/guards';
+import { GetOptions, IKeyValueManager, KeyValueData } from 'types/key-value-manager';
+import { isLocalEnvironment } from 'utils/env';
+import { Logger } from 'utils/logger';
+
+export abstract class KeyValueManager implements IKeyValueManager {
```

Comment:
this was the original implentation by Dor for the single env-var-manager. I thought it would be a good idea to keep it.

maybe it will be a better idea to explicitly state the child classes implement this interface?
i.e.
```
export class SecretsManager extends KeyValueManager implements IKeyValueManager {
  ....
```

Reply:
Yep, that's a cleaner pattern

Reply:
done